project/GrasshopperBuild.scala

```diff
@@ -141,6 +142,7 @@ object GrasshopperBuild extends Build {
         assemblyJarName in assembly := {s"grasshopper-${name.value}.jar"},
         assemblyMergeStrategy in assembly := {
           case "application.conf" => MergeStrategy.concat
+          case PathList("org", "joda", "time", "base", "BaseDateTime.class") => MergeStrategy.first
           case x =>
```

Comment:
See [this discussion](https://discuss.elastic.co/t/an-unshaded-joda-time-classes-in-elasticsearch-2-1-1-jar/38943) on why ES now provides its own version of joda-time.